ppr-ui/src/components/parties/EditDebtor.vue

```diff
@@ -0,0 +1,301 @@
+<template>
+  <div id="edit-debtor" class="white pa-6">
+    <v-expand-transition>
+      <v-row no-gutters>
+        <v-col cols="3">
+            <label
+              class="add-debtor-header general-label"
+              :class="{ 'error-text': invalidSection }"
+            >
+              <span v-if="activeIndex === -1" class="pl-5">Add</span>
+              <span v-else>Edit</span>
+              <span v-if="currentIsBusiness"> Business</span>
+              <span v-else> Individual Debtor</span>
+            </label>
+        </v-col>
+        <v-col cols="9">
+              <v-form
+                ref="debtorForm"
+                class="debtor-form"
+                v-on:submit.prevent="addDebtor"
+              >
+              <v-row v-if="currentIsBusiness" no-gutters>
+                    <v-col>
+                     <label class="general-label">Business Legal Name</label>
+                    </v-col>
+                </v-row>
+                <v-row v-else no-gutters>
+                    <v-col>
+                     <label class="general-label">Individual Name</label>
+                    </v-col>
+                </v-row>
+                <v-row v-if="currentIsBusiness" no-gutters>
+                    <v-col>
+                     <v-text-field
+                      filled
+                      id="txt-name"
+                      label="Business Legal Name"
+                      v-model="searchValue"
+                      :error-messages="errors.businessName.message ?
+                      errors.businessName.message : ''"
+                      persistent-hint
+                      :hide-details="hideDetails"
+                    />
+                    <auto-complete :searchValue="autoCompleteSearchValue"
+                       :setAutoCompleteIsActive="autoCompleteIsActive"
+                       @search-value="setSearchValue"
+                       @hide-details="setHideDetails">
+                    </auto-complete>
```

Comment:
not sure if this is just github formatting looking weird, but is the above block within this v-col indented?